fhir-model/src/main/java/com/ibm/fhir/model/path/evaluator/FHIRPathEvaluator.java

```diff
@@ -277,7 +288,7 @@ private void reset() {
             }
             Collection<FHIRPathNode> result = new ArrayList<>();
             ExpressionContext typeName = arguments.get(0);
-            String identifier = typeName.getText();
+            String identifier = typeName.getText().replace("`", "");
```

Comment:
duplicate line as the above... only commenting in case there is an explanation or change necessary.

Reply:
I just need to remove the tick if it is being used to escape an identifier. visitIdentifier has the same behavior.